lib/cuber/templates/deployment.yml.erb

```diff
@@ -194,16 +194,20 @@ spec:
           httpGet:
             port: 8080
             path: <%= URI.parse(@options[:health]).path.to_json %>
-            httpHeaders:
-              - name: Host
-                value: <%= URI.parse(@options[:health]).host.to_json %>
-              - name: X-Forwarded-Proto
-                value: <%= URI.parse(@options[:health]).scheme.to_json %>
           initialDelaySeconds: 0
-          periodSeconds: 10
+          periodSeconds: 1
```

Comment:
This would create a lot of health checks requests, usually 1 second, multiplied by all pods, are a lot of resources wasted, without any actual benefit compared to a 10s interval.